clients/privacy-center/config/config.json

```
@@ -8,11 +8,11 @@
     {
       "policy_key": "default_access_policy",
       "icon_path": "/download.svg",
-      "title": "Access your data",
-      "description": "We will provide you a report of all your personal data.",
+      "title": "Access your data (email or phone number)",
```

```suggestion
      "title": "Access your data",
```

This just looks odd as a default title

---

addressed